auth_track_and_prevent_brut_force/README.rst

```diff
@@ -0,0 +1,99 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+==============================================
+Authentication Tracking and Prevent Brut Force
+==============================================
+
+This module register each request done by users trying to authenticate into
+Odoo. If the authentication fails, a counter is increased for the given remote
+IP. After 10 attempts (or other quantity, depending on an ir.config_parameter),
+Odoo will ban the remote IP and ignore new requests.
+
+Admin user have the possibility to unblock a banned IP.
+
+This module realizes a call to a web API (http://ip-api.com) to try to have
+extra informations about remote IP.
+
+Note and Warning
+----------------
+The ID used to identify a remote request is the IP provided in the request
+(key 'REMOTE_ADDR').
+Depending of server and / or user network configuration, the idenfication
+of the user can be wrong, and mainly in the following cases:
+
+* if the Odoo server is behind an Apache / NGinx proxy without redirection,
+  all the request will be have the value '127.0.0.1' for the REMOTE_ADDR key;
+* If some users are behind the same Internet Service Provider, if a user is
+  banned, all the other users will be banned to;
+
+Logging
```

Comment:
Do you think a logging is needed when having an specific table for login attempts?